app/controllers/services_controller.rb

```diff
@@ -4,6 +4,7 @@ class ServicesController < ApplicationController
 
   def register
     changes_made = false
+    changed_service = nil
```

Comment:
Removing that causes failures when `changes_made` is true as well - we can't access a variable that doesn't exist in the current scope, so don't we still need to define it here?